### PR TITLE
Add Molnar's Triagonal-Agony

### PR DESCRIPTION
**Pull request by caa1211**
image

### CODE REVIEW

File: src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+     }
+}
+module pieces() {
+    burr_plate([
+        ["xx", "x.", "x."],
+        ["..x|xxx", "...|.x."],
+        ["xx|.x", "..|.x"],
+        ["x.|xx|.x", "..|..|.x"],
+        ["xx|.x", ".x|.."],
+        [".x|xx", "..|.x", "..|.x"]
```
Comment (aaron-siegel) [inline src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad]: Same comment as for L-I-Vator II. Would it be better to orient this piece (and the other height 3 piece on line 54) flat on the build plate?

File: src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+     difference() {
+        beveled_cube([dim, dim, dim]);
```
Comment (aaron-siegel) [inline src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad]: Random thought. What do you think about using a different, smaller bevel for the box? I tried this out when I did my print of Triagonal Agony. For example, we could change this line to

beveled_cube([dim, dim, dim], $burr_bevel = 0.5);

and make the similar change to the beveled_prism statement in triangle(). Using 0.5 would create a (visually) sharper box, and also reduces (though doesn't eliminate) the noticeable "gap" between the cap and box. [This is a purely stylistic question]

Also, note that if all the dimensions are the same, you can optionally use

beveled_cube(dim)

in place of

beveled_cube([dim, dim, dim]).

File: src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    render(convexity = 2)
+    rotate([0, -90, 0])
+    union() {
+        translate([0, box_puzzle_border, 0])
+        rotate([90, 0, 0])
+        triangle();
+
+        rotate([0, 0, 90])
+        translate([0, 0, dim - 2 * box_puzzle_border])
+        mirror([0,1,1])
```
Comment (aaron-siegel) [inline src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad]: code style: best to have spaces after the commas

**Review by aaron-siegel — APPROVED** [review]

**Comment by caa1211** [comment]
@aaron-siegel Updated, thanks

File: src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+module triangle() {
+   translate([dim - 2 * box_puzzle_border, 0, box_puzzle_border / 2])
+   rotate([90, 0, 0])
+   cylinder(r = pin_r, h = pin_height, $fn = 32);
+    
+   translate([box_puzzle_border / 2, 0, box_puzzle_border / 2])
+   rotate([90, 0, 0])
+   cylinder(r = pin_r, h = pin_height, $fn = 32);
+    
+   beveled_prism([[0, 0], [0, dim - 2 * box_puzzle_border], [box_puzzle_border, dim - 2 * box_puzzle_border], [dim - box_puzzle_border, 0]], height = box_puzzle_border);
```
Comment (aaron-siegel) [inline src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad]: I'd suggest adding $burr_bevel = 0.5 here as well if we're including it above

Comment (caa1211) [reply src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad]: Ok, updated

**Review by aaron-siegel — APPROVED** [review]

**Pull request merged**